src/test/java/org/glovo/service/OrderServiceTest.java

```diff
@@ -0,0 +1,163 @@
+package org.glovo.service;
+
+import org.glovo.entity.GlovoOrder;
+import org.glovo.entity.Product;
+import org.glovo.repository.OrderRepository;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.*;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.annotation.DirtiesContext;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
+public class OrderServiceTest {
+
+    private Product product1;
+    private Product product2;
+    private Product product3;
+    private List<Product> products;
+
+    public OrderServiceTest() {
+        product1 = new Product(1, "apple", 12.12, 2);
+        product2 = new Product(2, "nuts", 12, 2);
+        product3 = new Product(3, "banana", 12, 2);
+
+        products = new ArrayList<>();
+        products.add(product1);
+        products.add(product2);
+        products.add(product3);
+    }
+
+    @Mock
+    private OrderRepository orderRepository;
+
+    @Mock
+    private ProductService productService;
+
+    @InjectMocks
+    private OrderService orderService = new OrderService();
```

Comment:
We don't need create instance, if you use @InjectMocks it means that Spring create and get instance from context.